packages/common/src/operations.graphql

```diff
@@ -93,6 +93,58 @@ query invoice($id: String!) {
             numberOfPacks
             packSize
             note
+            location {
+              ... on LocationNode {
+                id
```

Comment:
the `__typename` isn't requested for `location` but is required by the `locationGuard` 
not sure how this is working?

Reply:
nice catch!